extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/KubernetesTaskRunnerTest.java

```diff
@@ -103,11 +112,30 @@ public KubernetesTaskRunnerTest()
         new NamedType(ParallelIndexTuningConfig.class, "index_parallel"),
         new NamedType(IndexTask.IndexTuningConfig.class, "index")
     );
+    this.useMultipleBaseTaskDirPaths = useMultipleBaseTaskDirPaths;
   }
 
-  @BeforeEach
-  void setUp()
+  @Parameterized.Parameters(name = "useMultipleBaseTaskDirPaths = {0}")
+  public static Collection<Object[]> getParameters()
   {
+    Object[][] parameters = new Object[][]{
+        {false},
+        {true}
+    };
+
+    return Arrays.asList(parameters);
+  }
+
+  @Before
+  public void setUp()
+  {
+    List<String> baseTaskDirPaths = null;
+    if (useMultipleBaseTaskDirPaths) {
+      baseTaskDirPaths = ImmutableList.of(
+          FileUtils.createTempDir().toString(),
```

Comment:
better to use `TemporaryFolder` in the test class and use that to create temp directory for test

Reply:
Done